Assets/BossRoom/Scripts/Client/Game/Character/CharacterSwap.cs

```diff
@@ -41,6 +42,25 @@ public void SetFullActive(bool isActive)
                 shoulderRight.SetActive(isActive);
                 shoulderLeft.SetActive(isActive);
             }
+
+            public List<GameObject> GetAllBodyParts()

```

Comment:
Are we able to serialize this? It will get called often.

Reply:
ok